Automattic/jetpack
# Debugger: Add package

## Changes proposed in this Pull Request:

* Moved the Debugger Library to a Debugger Package.

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* n/a

#### Testing instructions:
* Ensure the `wp jetpack status` command outputs as expected.
* Ensure the Site Health section (tests and data) report as expected.
* Ensure the jetpack.com debugger for SAs report local connection status as expected.

#### Proposed changelog entry for your changes:
* Debugger: Move functionality into an independent package.
